regression/jbmc-strings/StringIndexOf/test.desc

```diff
@@ -1,7 +1,14 @@
-CORE
+CORE symex-driven-lazy-loading-expected-failure
 Test.class
 --refine-strings --function Test.check --unwind 4 --string-max-input-length 3 --java-assume-inputs-non-null
 ^EXIT=0$
 ^SIGNAL=0$
 ^VERIFICATION SUCCESSFUL$
 --
+--
+Under symex-driven-lazy-loading the string solver seems to take a slightly different path,
+which leads to a very long Minisat run (without further intervention by the solver). I don't
+see any reason why this is the case except Minisat happening to get down a rabbit hole when
+there are other choices that lead to a quick solution. Disabling the test under symex-driven
+loading for now; most likely it will start working again in the near future for equally
+spurious reasons.
```

Comment:
Doesn't @romainbrenguier have some fix for indexOf? Is this maybe no longer relevant?

Reply:
Re-tested and this indeed appears to have cleared in the meantime. Removed the tag.